lib/tasks/client_generate.rake

```diff
@@ -39,7 +39,21 @@ class ClientGenerator
   end
 
   def generator_config
-    @generator_config ||= Pathname.new(Rails.root.join(".openapi_generator_config.json")).to_s
+    return @generator_config if @generator_config.present?
+
+    orig_config = Pathname.new(Rails.root.join(".openapi_generator_config.json")).to_s
+    json_settings = File.read(orig_config)
+
+    # Extend config by version used as client's Gem version
+    gem_version = "#{api_version}.0"
```

Comment:
You should be able to get the full api_version from the openapi file, we were actually revving the last component of the path so `.0` isn't always correct

Reply:
Do you mean from "info/version"? Which means I have to load and parse openapi.json file?